packages/ui/src/components/Tabs/utils.ts

```diff
@@ -39,6 +39,7 @@ export const commonTabStyles = {
     "data-h2-color": `
       base(black)
       base:children[a:focus-visible](black)
+      base:focus-visible:children[span](black)
```

Comment:
Still purple for me 😢 
![Screenshot 2024-01-26 112906](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/3fca3e3b-e9c2-4172-87ae-b13ff4a42263)

I think what you want is:
```suggestion
      base:focus-visible(black)
```

Reply:
Are you sure? It's definitely black on my end 😕

Reply:
That commit works too. Changed in: [dcf7d78](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9077/commits/dcf7d789cf4aaf4c3b103d40236f920a3798edc8)